tecken/tests/test_api.py

```diff
@@ -1177,35 +1177,3 @@ def test_file_upload_try_upload(client):
     assert response.status_code == 200
     data = response.json()["file"]
     assert data["url"] == "/foo.pdb/deadbeaf123/foo.sym?try"
-
-
-@pytest.mark.django_db
-def test_possible_upload_urls(client, settings):
```

Comment:
We still have this function, but it's a shade of it's former self and will likely get removed when I rewrite the storage layer. Ergo, I nixed this test.